public/js/app.js

```diff
@@ -0,0 +1,315 @@
+(function ($) {
+
+    $(document).ready(
+        function () {
+
+            /**************** Slider DEGRADE**********************/
+            var colors = new Array(
+[5, 4, 10],
+[33, 28, 66],
+[60, 51, 121],
+[88, 74, 176],
+[99, 86, 184],
+[238, 68, 55]);
+
+            var step = 0;
+            var colorIndices = [0, 1, 2, 3];
+
+            //transition speed
+            var gradientSpeed = 0.004;
+
+            function updateGradient() {
+
+                if ($ === undefined) return;
+
+                var c0_0 = colors[colorIndices[0]];
+                var c0_1 = colors[colorIndices[1]];
+                var c1_0 = colors[colorIndices[2]];
+                var c1_1 = colors[colorIndices[3]];
+
+                var istep = 1 - step;
+                var r1 = Math.round(istep * c0_0[0] + step * c0_1[0]);
+                var g1 = Math.round(istep * c0_0[1] + step * c0_1[1]);
+                var b1 = Math.round(istep * c0_0[2] + step * c0_1[2]);
+                var color1 = "rgb(" + r1 + "," + g1 + "," + b1 + ")";
+
+                var r2 = Math.round(istep * c1_0[0] + step * c1_1[0]);
+                var g2 = Math.round(istep * c1_0[1] + step * c1_1[1]);
+                var b2 = Math.round(istep * c1_0[2] + step * c1_1[2]);
+                var color2 = "rgb(" + r2 + "," + g2 + "," + b2 + ")";
+
+                $('.background').css({
+                    background: "-webkit-gradient(linear, left top, right top, from(" + color1 + "), to(" + color2 + "))"
+                }).css({
+                    background: "-moz-linear-gradient(left, " + color1 + " 0%, " + color2 + " 100%)"
+                });
+
+                step += gradientSpeed;
+                if (step >= 1) {
+                    step %= 1;
+                    colorIndices[0] = colorIndices[1];
+                    colorIndices[2] = colorIndices[3];
+
+                    colorIndices[1] = (colorIndices[1] + Math.floor(1 + Math.random() * (colors.length - 1))) % colors.length;
+                    colorIndices[3] = (colorIndices[3] + Math.floor(1 + Math.random() * (colors.length - 1))) % colors.length;
+                }
+            }
+
+            /**************** ScrollReveal *****************/
+            setInterval(updateGradient, 10);
+
+            window.sr = ScrollReveal({
+                reset: true
+            });
+            ScrollReveal().reveal('.headline', {
+                delay: 100
+            });
+            ScrollReveal().reveal('.tagline', {
+                delay: 500
+            });
+            ScrollReveal().reveal('.punchline', {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
            ScrollReveal().reveal(".punchline", {
```